### ISSUE TITLE
Long success messages can break mid-word

### ISSUE THREAD

**Issue by ravichandrasekaran** [juliasilge/supervised-ML-case-studies-course#30]
Long success messages can break mid-word, like this example from Chapter 4, section 11:
> Great job! In this case study, you will evaluate this model, plus two kinds of gradient boosting models.

I think one well-placed call to `stringr::str_wrap` would fix them all (and preserve the emoji's, hopefully), but can't find where to place it, or how to test it.

![c4-s11-word-break](https://user-images.githubusercontent.com/1671691/65784860-ec7ac400-e107-11e9-99b3-2626f2cde75c.png)

**Comment by juliasilge** [comment]
Yep, I noticed this as well and hadn't found a way to address it yet.

- The success messages are all in individual files called `test_02_11_2.R` and similar, and are [here in the repo](https://github.com/juliasilge/supervised-ML-case-studies-course/tree/master/exercises).
- The test files get used in [this component](https://github.com/juliasilge/supervised-ML-case-studies-course/blob/master/src/components/code.js) to generate the code exercises.
